packages/echo-core/src/types/components.ts

```diff
@@ -13,7 +13,7 @@ export type WrappedComponent<TProps> = ComponentType<Without<TProps, keyof BaseA
  * The props that every registered App component obtains.
  */
 export interface RouteBaseProps<UrlParams = unknown, UrlState = unknown>
-    extends RouteComponentProps<UrlParams, {}, UrlState>,
+    extends RouteComponentProps<UrlParams, Record<string, never>, UrlState>,
```

Comment:
Should this be never?

Reply:
Hmmm, no, I think it should be 'any object', so `Record<string, unknown>`. I try it out!